semantic_conventions/trace/database.yaml

```
@@ -402,6 +402,94 @@ groups:
           value MUST NOT be set.
         examples: ['public.users', 'customers']
 
+  - id: db.cosmosdb
+    type: span
+    extends: db
+    prefix: db.cosmosdb
+    brief: >
+      Call-level attributes for CosmosDb.
+    attributes:
+      - id: client_id
+        type: string
+        brief: Unique cosmos client instance id.
+        examples: '3ba4827d-4422-483f-b59f-85b74211c11d'
+      - id: operation_type
+        type:
+          allow_custom_values: false
```

please consider 

```suggestion
          allow_custom_values: true
```

Nothin really depends on someone following this list strictly, but setting the flag to `true` gives you more room to expend this list in future.